examples/nuxt-3/composables/checkout/index.ts

```diff
@@ -0,0 +1,66 @@
+import { get, set, del } from 'idb-keyval';
+import createShopifyCheckoutClient, {
+  CheckoutClient
+} from '@nacelle/shopify-checkout';
+import { ShopifyCheckout } from '@nacelle/shopify-checkout/dist/types/checkout-client.types';
+import createCheckout from '@nacelle/shopify-checkout/dist/types/client/actions/checkoutCreate';
```

Comment:
Looks like `createCheckout` is an unused import.

`ShopifyCheckout` is too at the moment, but it's used in another suggestion below.

```suggestion
```